src/Apps/Artwork/Components/ArtworkDetails/index.tsx

```diff
@@ -18,11 +20,49 @@ const ArtworkDetailsContainer = Box
 export class ArtworkDetails extends Component<ArtworkDetailsProps> {
   render() {
     const { artwork } = this.props
+    const renderAbout =
+      artwork.additional_information ||
+      artwork.description ||
+      artwork.framed ||
+      artwork.signatureInfo ||
+      artwork.conditionDescription ||
+      artwork.certificateOfAuthenticity ||
+      artwork.series ||
+      artwork.publisher ||
+      artwork.manufacturer ||
+      artwork.provenance ||
+      artwork.image_rights
```

Comment:
This is def #minor but when dealing with a lot of props like this its best to alpha (highlight, then cmd + shift + p > sort in VSCode) so in the future it will be easier to see what's there if we need to add / remove conditions.